.github/workflows/ci-coverage-build.yml

```diff
@@ -1,8 +1,6 @@
 name: Coverage Build
 on:
   workflow_dispatch:
-    branches:
-      - master
```

Comment:
why remove this? it's handy for me! :D 
does it disturb anything?

Reply:
my github linter in vscode tells me that it is not a valid key of workflow_dispatch ;) and I couldn't find it in the github docs. Isn't it that you can choose the branch for running the workflow anyways?

Reply:
pro'lly right, let's try removing it

Reply:
Can you run this workflow now manually for every branch?